plugin-ci/orb.yml

```diff
@@ -57,10 +74,10 @@ commands:
 aliases:
 - &restore_cache
   restore_cache:
-    key: go-mod-v1-{{ checksum "go.sum" }}
+    key: go-mod-v2-{{ checksum "go.sum" }}
 - &save_cache
   save_cache:
-    key: go-mod-v1-{{ checksum "go.sum" }}
+    key: go-mod-v2-{{ checksum "go.sum" }}
     paths:
     - "/go/pkg/mod"
```

Comment:
I think this needs an update https://github.com/CircleCI-Public/cimg-go/blob/48448c584a346740b4c4035292e06be1c14e04da/1.19/Dockerfile#L30